src/components/pages/EdcConnector/AddConnectorOverlay/index.tsx

```diff
@@ -247,7 +247,7 @@ const AddConnectorOverlay = ({
           {!loading && (
             <Button
               variant="contained"
-              disabled={selected?.id ? false : true}
+              disabled={selected?.id && isValid ? false : true}
```

Comment:
it is always better to go with ```true``` first and then ```false```

Reply:
Changes done